.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10"]
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11-dev"]
```

Comment:
Thanks for starting this. Looks like anaconda/miniconda doesn't support 3.11 yet:

https://repo.anaconda.com/pkgs/main/linux-64/